### PR TITLE
Update: Alexandria and UI/CSS fixes

### PR DESCRIPTION
**Pull request by omgaz**
- Update Alexandria (tags, cards with accent and append)
- SCSS formatting & variables in category search
- Export category search button icon as css data graphic and tighten designs
- Nudge react-select to latest beta

### CODE REVIEW

**Comment by patrick-ivers** [comment]
LGTM 🐙

File: src/components/Main.js [hunk 1]
```diff
@@ -708,7 +708,7 @@ class AppComponent extends React.Component {
           categoryOptions={this.state.categoryOptions}
           onCategorySelect={this.setCategory}
           searchString={this.state.categorySearchString}
-          searchPlaceholder="Search products"
+          searchPlaceholder="Search for all the things."
```
Comment (patrick-ivers) [inline src/components/Main.js]: 😐

**Comment by patrick-ivers** [comment]
LGTM take 2! :octocat:

File: src/styles/icon.scss [hunk 2]
```diff
@@ -5,6 +5,7 @@ $icon-folder-open: svg-url('<svg xmlns="http://www.w3.org/2000/svg" width="16" h
...
 $icon-arrow-bracket: svg-url('<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16"><path fill="#006dcc" d="M11 3L8.1 8l2.9 5H8.4L5 8.1v-.2L8.4 3H11z"/></svg>');
 $icon-chevron: svg-url('<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16"><path d="M1 8.6l6.8-4.9h.3L15 8.6v3.7L8 8.1l-7 4.2V8.6z"/></svg>');
+$icon-search: svg-url('<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16"><path fill="#fff" d="M14.8 12.8l-3.5-3.5c.9-1.6 1.1-3.6 0-5.6C10.5 2 8.7 1 6.8 1 4.1 1 1.7 2.8 1.1 5.4c-.7 3.6 2 6.7 5.4 6.7 1 0 2-.3 2.8-.8l3.4 3.4.1.1c.3.3.7.3.9 0l1.1-1.1c.3-.2.3-.7 0-.9zm-8.3-2.1c-2.9 0-5-2.9-3.7-5.9.6-1.5 2.2-2.4 3.7-2.4 1.1 0 2.1.4 2.9 1.2.8.8 1.2 1.8 1.2 2.9 0 1.1-.4 2.1-1.2 2.9-.7.9-1.8 1.3-2.9 1.3z"/></svg>');
```
Comment (OndrejCholeva) [inline src/styles/icon.scss]: `fill="#fff"` will make the icon hardcoded white :-/

**Pull request merged**